deploy/ansible/inventory/group_vars/all.yml

```diff
@@ -1,8 +1,18 @@
 ---
+torii_port: 50051
+internal_port: 10001
```

Comment:
In iroha/deploy/ansible/playbooks/iroha-docker-cluster/group_vars/all.yml and iroha/deploy/ansible/playbooks/iroha-docker-cluster/iroha-deploy.yml we specify 
```
  torii_port: 50051                 # torii port start value
  internal_port: 10001              # iroha port start value
```

but in iroha/deploy/ansible/inventory/group_vars/all.yml we already have
``` 
torii_port: 50051
internal_port: 10001
```

So why we duplicate this?

Reply:
@neewy they can be overwritten. it is called variables precedence. playbook's group vars overwrite intentory's group vars. that is very useful in case someone will want to change values in his self-written playbook

Reply:
Ok, good!
Can we add a single line comment that says these values overwrite group vars? :)